projects/plugins/protect/src/js/components/firewall-header/index.jsx

```diff
@@ -66,7 +66,24 @@ const UpgradePrompt = () => {
 	);
 };
 
-const FirewallHeader = ( { status, hasRequiredPlan } ) => {
+const CurrentlyEnabledFeatures = ( { manualRulesEnabled, status } ) => {
+	let enabledFeatures = '';
+
+	if ( status === 'on' ) {
+		if ( manualRulesEnabled ) {
+			enabledFeatures = __( 'Automatic and manual rules are currently active.', 'jetpack-protect' );
+		} else {
+			enabledFeatures = __( 'Automatic rules are currently active.', 'jetpack-protect' );
+		}
+	}
```

Comment:
Same situation here, as is the optimization is breaking the translation strings.

Reply:
Nice catch, fixed in 7dc7ad9a577